index.html

```diff
@@ -72,487 +43,6 @@ <h1 id="title">Coliseum Data Collection Tool (CODACOT)</h1>
       <div id="output_container"><div id="output_div"></div></div>
     </div>
   </div>
-
-  <script>
-    venues = [{
```

Comment:
This is brilliant, I love that you were able to get rid of so much of this spaghetti (something like 500 lines!)